edi_edifact_oca/components/party_data.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+
+
+class EDIPartyDataEDIFACT(Component):
+    """Party data provider specific to EDIFACT."""
+
+    _name = "edi.party.data.edifact"
+    _inherit = "edi.party.data"
+    _backend_type = "edifact"
```

Comment:
Can you add a TODO?  `TODO @simahawk: is a component per backend type really needed?`
I know you've taken it from edi_ubl_oca but we should try to remove special components when they are not needed IMO.